### DISCUSSION TITLE
How to unselect an item from the multi-select?

### DISCUSSION THREAD

**Discussion by Bipinoli** [helix-editor/helix#9378] [Q&A] [answered]
I can cycle through select items with `(` and `)` but don't know how to unselect a specific item.
How do you guys do that?

edit: for context. sometimes the search matches more words than I want. I want to be able to avoid changing the wrong words.

<img width="584" alt="image" src="https://github.com/helix-editor/helix/assets/11765482/13bb9aaa-071e-4fe8-bf52-1d74092ecbec">

**Comment by the-mikedavis** [comment]
You can use `A-,` (<kbd>Alt</kbd><kbd>,</kbd>) to remove the primary selection after rotating to the primary selection to the one you want to remove with `(`/`)`

**Reply by Bipinoli** [reply]
`A` takes me to the insert (append) mode immediately, `Alt,` does nothing and `/` just takes me to the search.
I can't seem to figure it out.

For context: I am on Macos with iterm2 but i tried other terminals without any success.

**Reply by the-mikedavis** [reply]
On a mac keyboard it would be holding Option and pressing `,`

**Reply by Bipinoli** [reply, marked as answer]
Thanks for the help @the-mikedavis 

Turns out the problem was the iterm2 setting.
It is working now after I changed the left option key from `normal` to `esc+` as in the picture below.


<img width="1277" alt="Screenshot 2024-01-18 at 22 24 16" src="https://github.com/helix-editor/helix/assets/11765482/103175e6-3ad9-4115-b34a-6665c0ab8e39">
